Makefile

```
@@ -100,15 +104,19 @@ tidy: #HELP Update dependencies.
 	$(Q)go mod tidy -go=$(GOLANG_VERSION)
 
 .PHONY: manifests
-manifests: $(CONTROLLER_GEN) #EXHELP Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/base/crd/bases output:rbac:artifacts:config=config/base/rbac
+# To exclude catalogd. Otherwise, the CRD, Webhook and other configurations will be generated and added to the config/
+PATHS = ./api/... ./cmd/... ./internal/... ./test/...
+manifests: $(CONTROLLER_GEN) #EXHELP Generate WebhookConfiguration, ClusterRole, and CustomResourceDefinition objects.
+	@for path in $(PATHS); do \
+		$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="$$path" output:crd:artifacts:config=config/base/crd/bases output:rbac:artifacts:config=config/base/rbac; \
+	done
 
 .PHONY: generate
 generate: $(CONTROLLER_GEN) #EXHELP Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

My suggestion from earlier is that we update this in much the same way that we handle the container build:
```suggestion
generate: $(CONTROLLER_GEN) #EXHELP Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
	$(MAKE) -C catalogd generate
```

That way we don't need the extra `generate-catalogd` target.

There's a separate wrinkle, which we would need to address:
- operator-controller has `generate` and `manifests` for both api codegen and manifest generation.
- catalogd has just `generate` that does both.

I think I'd suggest:
- remove `manifests` target
- move the `manifests` target's commands to `generate`
- have `generate` also invoke `catalogd`'s `generate`.

---

I see 👍 
Now I got what you mean: do not create the target, append the command.
Yep. That make sense I will do that

---

Solved